### DISCUSSION TITLE
Questions about the self hosted Cloud

### DISCUSSION THREAD

**Discussion by sigsdis30** [opengisch/QField#4497] [Q&A]
Hi,
I have a self hosted Cloud and I have a few questions :

What is the Django_cron section and what is it used for ?
In subscription : what is the "package_type" for ?
In Core, What are the "Deltas" used for ?
Is Geodbs the same as the secrets ?
What are the Groups used for ?

Thanks a lot for you help

**Comment by suricactus** [comment]
Feel free to ask for dedicated support at sales@opengis.ch .
